src/components/dataDisplays/DataDisplay.jsx

```diff
@@ -104,7 +104,12 @@ export default function DataDisplay({
               .trim()
           : JSON.stringify(rawValue);
       }
-      if (dataValue && dataValue.includes(filter)) match = true;
+      if (
+        dataValue &&
+        (dataValue.includes(filter) ||
+          dataValue.includes(filter.toLowerCase()))
+      )
```

Comment:
Since dataValue is lowercased above, `dataValue.includes(filter.toLowerCase()))` must be true whenever `dataValue.includes(filter)` is true. So you can simplify:

```
if (
        dataValue && dataValue.includes(filter.toLowerCase()))
      )
```